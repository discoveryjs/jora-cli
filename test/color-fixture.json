{
    "string": "st\"ri\u1234ng",
    "number": 1234,
    "emptyArray": [],
    "emptyObject": {},
    "singlePropObject": { "foo": "test" },
    "null": null,
    "false": false,
    "true": true,
    "complexJSON": {
        "f\"oo\u01234"
            :-0.1234, "bar": 0e-3,
        "ba/": -0.1234e12,
        "quux": -1.123
    }
}
