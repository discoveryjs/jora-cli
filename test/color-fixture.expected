[90m{
    "[39mstring[90m": [32m"st\"riáˆ´ng"[90m,
    "[39mnumber[90m": [36m1234[90m,
    "[39memptyArray[90m": [],
    "[39memptyObject[90m": {},
    "[39msinglePropObject[90m": {
        "[39mfoo[90m": [32m"test"
    [90m},
    "[39mnull[90m": [39m[1mnull[22m[90m,
    "[39mfalse[90m": [36mfalse[90m,
    "[39mtrue[90m": [36mtrue[90m,
    "[39mcomplexJSON[90m": {
        "[39mf\"ooÄ£4[90m": [36m-0.1234[90m,
        "[39mbar[90m": [36m0[90m,
        "[39mba/[90m": [36m-123400000000[90m,
        "[39mquux[90m": [36m-1.123
    [90m}
}[39m
